frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/disk-wrapper.ts

```diff
@@ -25,7 +25,7 @@ export class DiskWrapper extends ObjectWithTypePropertyWrapper<V1Disk, DiskType>
       },
       {
         initializeWithType: type,
-        initializeWithTypeData: type === DiskType.DISK && bus ? { bus: bus.getValue() } : undefined,
+        initializeWithTypeData: bus ? { bus: bus.getValue() } : undefined,
```

Comment:
it should rather say following, because the bus is defined for everything except floppies

```suggestion
        initializeWithTypeData: type !== DiskType.FLOPPY && bus ? { bus: bus.getValue() } : undefined,
```

Reply:
Floppies are deprecated no?   
https://kubevirt.io/user-guide/docs/latest/creating-virtual-machines/disks-and-volumes.html#floppy

Reply:
ah, I see, thanks